src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -226,6 +227,99 @@ private function addProfilerSection(ArrayNodeDefinition $rootNode)
         ;
     }
 
+    private function addWorkflowSection(ArrayNodeDefinition $rootNode)
+    {
+        $rootNode
+            ->children()
+                ->arrayNode('workflows')
+                    ->useAttributeAsKey('name')
+                    ->prototype('array')
+                        ->children()
+                            ->arrayNode('marking_store')
+                                ->isRequired()
+                                ->children()
+                                    ->enumNode('type')
+                                        ->values(['property_accessor', 'scalar'])
```

Comment:
The code mixes the short array notation and the traditional array notation. IIRC, only the traditional notation is allowed, even for modern Symfony versions.